templates/chrony.conf.epp

```diff
@@ -118,11 +135,11 @@ noclientlog
 clientloglimit <%= $chrony::clientloglimit %>
 <% } -%>
 
-# Send a message to syslog if a clock adjustment is larger than 0.5 seconds.
-logchange 0.5
+# Send a message to syslog if a clock adjustment is larger than specified threshold
+logchange <%= $chrony::logchange %>
 <% if $chrony::mailonchange { -%>
 
-# Send mail  if chronyd applied a correction exceeding given threshold.
+# Send mail if chronyd applied a correction exceeding given threshold.
```

Comment:
Although I agree, it's untidy, changing comments will cause every single instance to restart, which is disruptive, not a big fan

Reply:
Should I put the space back?

Reply:
Since the template is also updated by the commit I would keep the fix.  I mean people will see change when they update the module, and they can expect a service restart IMHO.

Reply:
I am less concerned about this comment, since it's in conditional block and not enabled by default.
Changing comment above logchange will affect everyone

Reply:
Indeed and the current value kinda sucks :-)  Expanding the variable in the comment looks terrible, so I like the way it is fixed by rephrasing the comment.  Wes it will change the config file of everybody, but for the best… so it's a win ?

Reply:
I think this is fine now. Does one of you have objections?

Reply:
Restarting chrony isn't a massive deal IMO.